src/App.svelte

```diff
@@ -20,10 +47,11 @@
   }
 
   page("/", setComponent(AppList));
-  page("/apps/:app/tables/:ping", setComponent(TableDetail));
+  page("/apps/:app/tables/:ping/:bigquery", setComponent(TableDetail));
```

Comment:
This should actually be `:table` (using `:ping` was actually a mistake)

```suggestion
  page("/apps/:app/tables/:table", setComponent(TableDetail));
```

Reply:
Note to do this you'll also need to make some (very minor) changes to the table component: https://github.com/mozilla/glean-dictionary/blob/9990cef3e5e04a06aaeb6d69470516325e5e8535/src/pages/TableDetail.svelte#L6